locale/manageiq.pot

```diff
@@ -4388,6 +4387,10 @@ msgstr ""
 msgid "Add group"
 msgstr ""
 
+#: ../app/javascript/components/AeInlineMethod/index.jsx
+msgid "Add method"
+msgstr ""
```

Comment:
@GilbertCherrie can you take a look and see if someone can take a look at the msgid field here.  It looks like we have duplicate lines with different casing.  See above.  I assume we can normalize to a single form.

Reply:
I fixed it in this pr: https://github.com/ManageIQ/manageiq-ui-classic/pull/9242